metals/src/main/scala/scala/meta/internal/metals/MtagsResolver.scala

```diff
@@ -90,7 +90,9 @@ object MtagsResolver {
           case State.Success(v) => Some(v)
           // Try to download latest supported snapshot
           case _: State.Failure
-              if original.isEmpty && scalaVersion.contains("NIGHTLY") =>
+              if original.isEmpty &&
+                scalaVersion.contains("NIGHTLY") ||
+                scalaVersion.contains("nonbootstrapped") =>
```

Comment:
Hopefully no one minds since it's a super small change and impacts miniscule amount of people.